### PR TITLE
JS testing section

### PR DESCRIPTION
**Pull request by reginafcompton**
@hancush - ready for feedback, when you return to the States next week!

### CODE REVIEW

File: intro-to-javascript-testing.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
+# Intro to JavaScript Testing
+
+* [The right style](#style)
```
Comment (hancush) [inline intro-to-javascript-testing.md]: This link should be `#the-right-style` ;-)

File: intro-to-javascript-testing.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+* [Test your syntax with jshint](#jshint)
+* [Test your logic with jasmine](#jasmine)
+
+## The right style
+
+JavaScript, as a language itself, does not have too many opinions. Functional, reliable, "correct" JS comes in a variety of colors and shapes: in fact, the number of ways to [declare a function](https://www.bryanbraun.com/2014/11/27/every-possible-way-to-define-a-javascript-function/) reaches double digits.
```
Comment (hancush) [inline intro-to-javascript-testing.md]: Awesome example!

File: intro-to-javascript-testing.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+
+* [The right style](#style)
+* [Test your syntax with jshint](#jshint)
+* [Test your logic with jasmine](#jasmine)
+
+## The right style
+
+JavaScript, as a language itself, does not have too many opinions. Functional, reliable, "correct" JS comes in a variety of colors and shapes: in fact, the number of ways to [declare a function](https://www.bryanbraun.com/2014/11/27/every-possible-way-to-define-a-javascript-function/) reaches double digits.  
+
+Its flexibility can cause confusion, headaches, and pure rage. JS invites hard-to-answer questions, like, "Am I doing this right?", "Am I doing this the best way possible?" – and the "right way" may encompass enough solutions to ruin an afternoon.
```
Comment (hancush) [inline intro-to-javascript-testing.md]: I'd tack the "pure rage" sentence on to the end of the previous paragraph and scrap the rest of this one.

File: intro-to-javascript-testing.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+
+...
+
+script: 
+- pytest tests
+- jshint my_unique_app/static/js/custom.js
+
+...
+``` 
+
```
Comment (hancush) [inline intro-to-javascript-testing.md]: Perhaps we should also mention that you can customize [what rules the `jshint` CLI enforces](http://jshint.com/docs/options/) in a `.jshintrc` file?

Comment (hancush) [reply intro-to-javascript-testing.md]: (Even better, perhaps we ought to make a standard DataMade-flavored `.jshintrc`!)

File: intro-to-javascript-testing.md [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+...
+```
+
+Then, in terminal, run:
+
+```bash
+jasmine
+```
+
+Visit `http://127.0.0.1:8888`, and view your test results.
```
Comment (hancush) [inline intro-to-javascript-testing.md]: Do you have to view Jasmine output in a browser?

Comment (reginafcompton) [reply intro-to-javascript-testing.md]: I do not think so...but I'd like to open a Jasmine-specific issue about this and other questions.

File: intro-to-javascript-testing.md (hunk 4 above)
Comment (reginafcompton) [reply intro-to-javascript-testing.md]: Oh, yes!

**Pull request merged**